codec/src/main/java/io/netty/handler/codec/DefaultHeaders.java

```diff
@@ -603,7 +603,11 @@ public T clear() {
     @Override
     public Boolean getBoolean(K name) {
         V v = get(name);
-        return v != null ? valueConverter.convertToBoolean(v) : null;
+        try {
+            return v != null ? valueConverter.convertToBoolean(v) : null;
+        } catch (Exception ignore) {
```

Comment:
nit: RuntimeException for clarity, although it is equivalent to what's here.

Reply:
@ejona86 We are talking about all the `catch` correct ? If so I think that makes sense @carl-mastrangelo @Scottmitch WDYT ?

Reply:
Yeah, I was talking about all of them

Reply:
`RuntimeException` sgtm